File: app/src/main/java/com/galib/natorepbs2/db/ComplainCentre.java
package com.galib.natorepbs2.db;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "complain_center_table")
public class ComplainCentre {
    @PrimaryKey
    @NonNull
    @ColumnInfo(name = "name")
    String name;
    @NonNull
    @ColumnInfo(name = "mobile_no")
    String mobileNo;
    @ColumnInfo(name="priority")
    int priority;
    public ComplainCentre(int priority, @NonNull String name, @NonNull String mobileNo){
        this.name = name;
        this.mobileNo = mobileNo;
        this.priority = priority;
    }
    @NonNull
    public String getName() {
        return name;
    }

    @NonNull
    public String getMobileNo() {
        return mobileNo;
    }
}

File: app/src/main/java/com/galib/natorepbs2/models/ComplainCentre.kt
package com.galib.natorepbs2.models

import androidx.room.PrimaryKey
import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity(tableName = "complain_center_table")
class ComplainCentre(
    @ColumnInfo(name = "priority") var priority: Int,
    @ColumnInfo(name = "name") @PrimaryKey var name: String,
    @ColumnInfo(name = "address") var address:String,
    @ColumnInfo(name = "mobile_no") var mobileNo: String,
    @ColumnInfo(name = "telephone_no") var telephoneNo: String,
    @ColumnInfo(name = "email") var email: String,
    @ColumnInfo(name = "latitude") var latitude: Double,
    @ColumnInfo(name = "longitude") var longitude: Double
)